content/localdotcom/drone-azure-pipelines/index.md

```diff
@@ -0,0 +1,116 @@
+---
+version: '1.0.0'
+date: 2020-01-20T00:00:00+00:00
+title: Azure Pipelines
+author: localdotcom
+tags: [ infrastructure, ops, azure]
+logo: azure_pipelines.svg
+repo: localdotcom/drone-azure-pipelines
+image: dclocal/drone-azure-pipelines:1.0.0
+---
+
+This plugin can be used to bulk run Azure pipelines. The below sample pipeline configuration demonstrates simple usage:
+
+```yaml
+kind: pipeline
+name: Bulk run Azure pipelines
+
+platform:
+  os: linux
+  arch: amd64
+
+steps:
+- name: Run build pipeline
+  image: dclocal/drone-azure-pipelines:1.0.0
+  settings:
+    action: queue_build
+    definitions: build_definition_id
+      - build_definition_id
+    organization: organization
+    project: project
+    user: username@domain.com
+    secret:
+      from_secret: az_secret
+
+- name: Create a new release
+  image: dclocal/drone-azure-pipelines:1.0.0
+  settings:
+    action: release
+    definitions: release_definition_id
+    stages: development
+    organization: organization
+    project: project
+    user: username@domain.com
+    secret:
+      from_secret: az_secret
+```
+
+Example configuration with multiple build pipelines:
+```yaml

```

Comment:
An an empty line before this block.

Reply:
Empty line removed.

Reply:
There was a small typo in the message, you should add an empty line around the "headlines".